web/core/components/global/product-updates/jsxConverter.tsx

```
@@ -0,0 +1,274 @@
+        console.warn("Node for 'colored-text' inlineBlock is missing 'text' field:", node);
+        return null;
+      }
+
+      return <span className="text-blue-500 font-bold font-mono break-all">{text}</span>;
+    },
+  },
+};
+
+const UploadJSXConverter: JSXConverters<any> = {
+  upload: ({ node }) => {
+    if (node.value && node.value.url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)